src/test/java/com/znsio/api/typicode.feature

```diff
@@ -0,0 +1,34 @@
+@prod
+Feature: Fetch User Posts and Albums
+
+  Background:
+    * def userId = 1
+    * def invalidUserId = 999
+
+  @post
+  Scenario: User Should be able to Fetch all Posts of UserId 1.
+    Given def userPosts = karate.call('classpath:com/znsio/templates/typicodeTemplates.feature@t_getPosts', { 'userId' : userId }).response
+    Then karate.log("User Posts Length: " + userPosts.length)
+    And assert userPosts.length >= 1
+    And match each userPosts..userId == 1
+    And match userPosts[0] == read('classpath:com/znsio/templates/schema.json').expectedPostsResponse
```

Comment:
Improve readability of code,  define this above read('classpath:com/znsio/templates/schema.json') and then call its variable.